src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/MergeSigningInfo.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Microsoft.DotNet.Maestro.Tasks.Tests
+{
+    [TestFixture]
+    public class MergeSigningInfo
+    {
+        private PushMetadataToBuildAssetRegistry pushMetadata;
```

Comment:
What does this object do if it doesn't have any state and no dependencies (it's constructed with an empty constructor).  It's a little weird they aren't static methods...

Reply:
That is an excellent question. As far as I can tell most (or all) of these methods could be static, but aren't. Is it because it's an MSBuild task?

Reply:
I am also chasing some phantom state that I believe is in my test code (since I switched to using a SetUp method to initialize this it really shouldn't be state in the class), but since it's still phantom I could have missed something else I guess.

Reply:
Phantom state was a product bug, unrelated to this. However, the first answer still stands - the methods aren't static so I need an instance.